tools/defaults.bzl

```diff
@@ -68,7 +68,34 @@ def ng_test_library(deps = [], tsconfig = None, **kwargs):
     **kwargs
   )
 
-def ng_web_test_suite(deps = [], srcs = [], **kwargs):
+def ng_web_test_suite(deps = [], srcs = [], static_css = [], **kwargs):
+  # Workaround for https://github.com/bazelbuild/rules_typescript/issues/301
+  # Since some of our tests depend on CSS files which are not part of the `ng_module` rule,
+  # we need to somehow load static CSS files within Karma (e.g. overlay prebuilt). Those styles
+  # are required for successful test runs. Since the `ts_web_test_suite` rule currently only
+  # allows JS files to be included and served within Karma, we need to create a JS file that
+  # loads the given CSS file.
+  for css_label in static_css:
+    css_id = "static-css-file-%s" % (css_label.strip(':').strip('/'))
+    deps += [":%s" % css_id]
+
+    native.genrule(
+      name = css_id,
+      srcs = [css_label],
+      outs = ["%s.js" % css_id],
+      output_to_bindir = True,
+      cmd = """
+        files=($(locations %s))
+        css_content=$$(cat $${files[0]})
+        js_template="var cssElement = document.createElement('style'); \
+                    cssElement.type = 'text/css'; \
+                    cssElement.innerHTML = '$$css_content'; \
+                    document.head.appendChild(cssElement);"
```

Comment:
Could this be a separate file that's included in the rule rather than creating it on-demand with a `genrule`?

Reply:
Not entirely sure what you mean? A separate rule for generating this file?

Not sure if that would be worth the effort.

This is just a matrix function, so we don't have access to a context.

Reply:
I mean creating a real js file like `overlay-test-style-include.js` that's added to the deps of just the overlay rule?

Reply:
I don't think that will work. I'm not sure how we could load the prebuilt CSS output into it without a custom rule/or genrule

Reply:
Ah, I see now.